modules/util/portal-tools-rest-builder-test-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,18 @@
 components:
     schemas:
+        BatchTestEntity:
```

Comment:
The only think I would change is renaming from `BatchTestEntity` to `CompanyBatchTestEntity`, to make it match with the existing pattern. But it is also true we don't have any other scope yet, so I think this is ok for now